telco-examples/mgmt-cluster/eib/custom/scripts/install-dependencies.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+mount /usr/local || true
+mount /home || true
+
+## create folder to server httpd media server
+mkdir -p /home/metal3/bmh-image-cache
+
+## copy the metal3 yaml file to metal3 folder
+cp ./helm-values-metal3.yaml /home/metal3/
+cp ./clusterctl.yaml /home/metal3/
+
+# Create the installer script
+cat <<- EOF > /usr/local/bin/mgmt-cluster-installer.sh
+#!/bin/bash
+set -euo pipefail
+
+## install clusterctl and helm
+mount /usr/local || true
```

Comment:
This shouldn't be necessary. We have already properly booted here.

Reply:
ok Thanks